pkg/ddc/alluxio/load_data.go

```diff
@@ -131,7 +136,7 @@ func (e *AlluxioEngine) generateDataLoadValueFile(r cruntime.ReconcileRequestCon
 	if err != nil {
 		return
 	}
-	err = os.WriteFile(valueFile.Name(), data, 0400)
+	err = os.WriteFile(valueFile.Name(), data, 0o400)
```

Comment:
What's the difference between 0400 and 0o400?

Reply:
Go 1.13 adds the 0o form for octal numbers.  we can write 0o644 instead of writing 0644 
[Go 1.13 Release Notes](https://tip.golang.org/doc/go1.13#gofmt)

both ways are supported, personal habit. this is automatically modified by my VSCode

Reply:
Ok. got it. Thx